tests/modules/asset_groups/resources/test_create_asset_group.py

```diff
@@ -110,6 +111,17 @@ def test_create_asset_group(flask_app_client, researcher_1, readonly_user, test_
             flask_app_client, researcher_1, data.get(), 400, resp_msg
         )
         data.content['sightings'][0].pop('ambiguity')
+        resp_msg = 'locationId is not a valid uuid string in Encounter 1.1'
+
+        data.set_encounter_field(0, 0, 'locationId', 'Enceladus')
+        asset_group_utils.create_asset_group(
+            flask_app_client, researcher_1, data.get(), 400, resp_msg
+        )
+        data.set_encounter_field(0, 0, 'locationId', str(uuid.uuid4))
```

Comment:
i think this is meant to be `uuid.uuid4()` ... though setting `locationId` as the function itself probably is just as invalid?  heh

Reply:
Have a look at the (now) wrong ones that I removed from the other files, It needs to be passed as a str

Reply:
i meant the inner part.  e.g. should this be: `str(uuid.uuid4())` ?   (note parens)